cactusbot/handlers/events.py

```diff
@@ -23,8 +26,20 @@ def on_follow_return():
 
         if packet.success:
             if self.cache_follows:
-                if not(CacheUtils.in_cache_followers(packet.user)):
-                    CacheUtils.cache_followers_add(packet.user)
+                cache = CacheUtils("caches/followers.json")
+                if cache.in_cache(packet.user):
+                    if self.cache_follows_time > 0:
+                        configtime = int(self.cache_follows_time * 60)
+                        cache_time = int(calendar.timegm(tuple(
```

Comment:
If we store the seconds-since-epoch instead of the `time_struct`, we can just use `cache.return_data(packet.user)` here, which would be much cleaner.